modules/bmo-about-the-firmwareschema-resource.adoc

```diff
@@ -0,0 +1,45 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_content-type: REFERENCE
+[id="about-the-firmwareschema-resource_{context}"]
+= About the FirmwareSchema resource
+
+BIOS settings vary among hardware vendors and host models. A `FirmwareSchema` resource is a read-only resource that contains the types and limits for each BIOS setting on each host model. The data comes directly from the BMC through Ironic. The `FirmwareSchema` prevents you from sending invalid values to a host when specifying values in the `spec` field of the `HostFirmwareSettings` resource. The `FirmwareSchema` resource has a unique identifier derived from its settings and limits. Identical host models use the same `FirmwareSchema` identifier. It is likely that multiple instances of `HostFirmwareSettings` use the same `FirmwareSchema`.
```

Comment:
IMHO, `The FirmwareSchema prevents you from sending invalid values to a host when specifying values in the spec field of the HostFirmwareSettings resource` is confusing. @bfournie I'm not sure the schema is preventing something. I thought is just information. WDYT?